Cargo.toml

```diff
@@ -1,39 +1,16 @@
 [package]
 name = "hyperjson"
-version = "0.2.2"
+version = "0.2.1"
```

Comment:
We're at 0.2.2 and I kinda got used to the extra metadata fields in the `Cargo.toml`. 😉 Do you think we should roll back the changes in this file?